### PR TITLE
va_display_drm: Allow VGEM for WSL2

### PR DESCRIPTION
**Pull request by sivileri**
Signed-off-by: Sil Vilerino <sivileri@microsoft.com>

Related https://github.com/intel/libva/pull/688

### CODE REVIEW

**Comment by sivileri** [comment]
@dvrogozh this change makes `vainfo --display drm` work along with the recently approved https://github.com/intel/libva/pull/688

File: common/va_display_drm.c [hunk 1]
```diff
@@ -81,7 +82,10 @@ va_open_display_drm(void)
             close(drm_fd);
             continue;
         }
-        if (!strncmp(version->name, "vgem", 4)) {
+        struct utsname sysinfo = {};
+        if (!strncmp(version->name, "vgem", 4)
```
Comment (dvrogozh) [inline common/va_display_drm.c:86]: As suggested by @evelikov in other review:
```
if (!strncmp(version->name, "vgem", 4) && (uname(&sysinfo) >= 0) &&
    !strstr(sysinfo.release, "WSL")) {
```

**Review by dvrogozh — CHANGES_REQUESTED** [review]

**Review by dvrogozh — APPROVED** [review]
looks good

File: common/va_display_drm.c [hunk 2]
```diff
@@ -81,7 +82,9 @@ va_open_display_drm(void)
             close(drm_fd);
             continue;
         }
-        if (!strncmp(version->name, "vgem", 4)) {
```
Comment (evelikov) [inline common/va_display_drm.c:84]: Do you mind adding a small comment:
On normal Linux platforms we do not want vgem. Yet WSL uses vgem, while also providing a fallback VA driver (see libva for more)

**Review by XinfengZhang — APPROVED** [review]

**Pull request merged**